handlers/istio_config.go

```diff
@@ -136,6 +138,12 @@ func parseCriteria(namespace string, objects string) business.IstioConfigCriteri
 	if checkType(types, business.Policies) {
 		criteria.IncludePolicies = true
 	}
+
+	// MeshPolicies are not namespaced. They will be only listed for the namespace
+	// where istio is deployed.
+	if checkType(types, business.MeshPolicies) && namespace == config.Get().IstioNamespace {
```

Comment:
I guess this filter should be done in a inner layer (business).
Handler should propagate the query with IncludeMeshPolicies = true if the param has been passed to the request.